tests/test_doctests.py

```diff
@@ -57,39 +59,46 @@
-suite.addTest(doctest.DocFileSuite('tests/nif/fix_detachhavoktristripsdata.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/fix_clampmaterialalpha.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_mergeduplicates.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/modify_delbranches.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/modify_delvertexcolor.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/fix_cleanstringpalette.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/modify_substitutestringpalette.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/modify_allbonepriorities.txt'))
-suite.addTest(doctest.DocFileSuite('tests/om_simpletype.txt'))
-suite.addTest(doctest.DocFileSuite('tests/om_arraytype.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/matrix.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/skinpartition.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/bhkpackednitristripsshape.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_delunusedbones.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_delzeroscale.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_collisiongeometry.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_collision_to_box_shape.txt'))
-suite.addTest(doctest.DocFileSuite('tests/nif/opt_vertex_cache.txt'))
-suite.addTest(doctest.DocFileSuite('tests/cgf/cgftoaster.txt'))
-suite.addTest(doctest.DocFileSuite('tests/kfm/kfmtoaster.txt'))
-suite.addTest(doctest.DocFileSuite('docs-sphinx/intro.rst'))
+filepaths = { 'object_model/simpletype.txt',
+              'object_model/arraytype.txt',
+              #'cgf/cgftoaster.txt',
+             # 'tests/nif/niftoaster.txt',
+             # 'tests/nif/optimize.txt',
+             # 'tests/nif/dump_tex.txt',
+             # 'tests/nif/ffvt3rskin.txt',
+             # 'tests/nif/fix_texturepath.txt',
+             # 'tests/nif/fix_tangentspace.txt',
+             # 'tests/nif/fix_tangentspace_series_parallel.txt',
+             # 'tests/nif/fix_detachhavoktristripsdata.txt',
+             # 'tests/nif/fix_clampmaterialalpha.txt',
+             # 'tests/nif/opt_mergeduplicates.txt',
+             # 'tests/nif/modify_delbranches.txt',
+             # 'tests/nif/modify_delbranches.txt',
+             # 'tests/nif/modify_delvertexcolor.txt',
+             # 'tests/nif/fix_cleanstringpalette.txt',
+             # 'tests/nif/modify_substitutestringpalette.txt',
+             # 'tests/nif/modify_allbonepriorities.txt',
+             # 'tests/nif/matrix.txt',
+             # 'tests/nif/skinpartition.txt',
+             # 'tests/nif/bhkpackednitristripsshape.txt',
+             # 'tests/nif/opt_delunusedbones.txt',
+             # 'tests/nif/opt_delzeroscale.txt',
+             # 'tests/nif/opt_collisiongeometry.txt',
+             # 'tests/nif/opt_collision_to_box_shape.txt',
+             # 'tests/nif/opt_vertex_cache.txt',
+
+             # 'tests/kfm/kfmtoaster.txt',
+             # 'docs-sphinx/intro.rst',
```

Comment:
Yes, same issues as previous as this is a way to externally define doctests, but still have issues with pathing.